TCA Counter App/Root/RootView.swift

```diff
@@ -10,6 +10,7 @@ import ComposableArchitecture
 
 struct ContentView: View {
     let store: Store<RootState, RootAction>
+    @State private var showPresent = false
```

Comment:
In general, I would say while the TCA alternative to this is boiler platy, it would be the preferred approach i.e. move this variable to the `RootState` and derive a binding from the view store to handle the sheet presentation. That is basically because if we do it this way, we can't cover the state changes corresponding to the sheet presentation during unit tests.

Reply:
@nanocba Thanks a lot,  I refactored it once based on your suggestion